miso-web/src/main/webapp/pages/editStudy.jsp

```diff
@@ -89,8 +89,15 @@
       <tr>
         <td class="h">Project Name:</td>
         <td>
-          <input type="hidden" value="${study.project.id}" name="project" id="project"/>
-          <a href='<c:url value="/miso/project/${study.project.id}"/>'><span id="projectName">${study.project.name}</span></a>
+        <c:choose>
+          <c:when test="${not empty study.project}">
+            <input type="hidden" value="${study.project.id}" name="project" id="project"/>
+            <a href='<c:url value="/miso/project/${study.project.id}"/>'><span id="projectName">${study.project.name}</span></a>
```

Comment:
Most of our users (detailed sample mode) know the project by its shortname (but in plain sample mode, the shortname can be left blank). Could the dropdown menu be updated to display the project's shortname if it is detailed sample mode, and the project name if it's plain sample mode?

Reply:
What distinguishes the plain and detailed sample modes?